src/overlays/actors/ovl_Bg_Po_Syokudai/z_bg_po_syokudai.h

```diff
@@ -8,7 +8,11 @@ struct BgPoSyokudai;
 
 typedef struct BgPoSyokudai {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x64];
+    /* 0x014C */ u8 unk_14C;
+    /* 0x014E */ s16 unk_14E;
```

Comment:
unk_14C is the Poe sister the torch corresponds to and unk_14E is the scroll for the flame texture, so these can be named accordingly.